tests/test_config.py

```diff
@@ -200,3 +203,55 @@ def test_remediation_no_results(self):
         p.expand_variables(p.get_variables())
         resources = p.run()
         self.assertEqual(len(resources), 0)
+
+    def test_retention_configurations(self):
+        session_factory = self.replay_flight_data('test_retention_configurations')
```

Comment:
cool, fwiw to get the files, use record_flight_data, we anonymize aws account ids, in the recorded data, and then when files are on disk, switch to replay.

https://cloudcustodian.io/docs/developer/tests.html#recording-custodian-interactions